### ISSUE TITLE
Breaks when i use Wand/Wand doesnt work in multiplayer

### ISSUE THREAD

**Issue by OrionDragonstar** [The-Fireplace/Overlord#244]
### Mod Version

3.2.3

### What mod loader(s) are you seeing the problem on?

Forge

### Mod Loader Version

43.1.7

### Problem Description

Controller Wand Doesn't work in multiplayer, The UI Doesn't Show, In Single player The Ui shows up but if you click on anything it crashes the game, Creating Squads just doesn't work, gets stuck saving

### Steps To Reproduce

Get wand in Multiplayer
Use Wand
Cry because the Ui wont appear
---------------------------
Get wand in Singleplayer
Use Wand
Ui Will Appear
Click on anything
Cry Because Your Game Crashed

### Crash Report or Error Logs

```shell
The game crashed whilst mouseclicked event handler
Error: java.util.NoSuchElementException: No value present -1
```

**Comment by OrionDragonstar** [comment]
[crash-2022-12-03_19.02.04-client.txt](https://github.com/The-Fireplace-Minecraft-Mods/Overlord/files/10147575/crash-2022-12-03_19.02.04-client.txt)

**Comment by Knightaa123** [comment]
I have this same wand problem, but in the single player it is already crashing and it's in the fabric version

**Comment by The-Fireplace** [comment]
@Knightaa123 Please make a separate issue, I'm not getting a crash in the Fabric builds of the mod.

**Comment by LeetleMay** [comment]
yeah, forge overlord 3.2.3 doesn't work in multiplayer and neither in solo or multi can you create a squad.

**Comment by The-Fireplace** [comment]
Fixed in Fireplace Lib 7.1.0
